package.json

```diff
@@ -46,10 +46,10 @@
     "three": ">=0.133"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.14.1592",
+    "@nuxt/kit": "^3.16.0",
     "@nuxt/ui": "^2.20.0",
     "@tresjs/core": "4.3.1",
-    "@unocss/nuxt": "^0.65.2",
+    "@unocss/nuxt": "^66.1.2",
```

Comment:
same here

Reply:
I moved it down to dev dependencies, on the house..